products/caregivers/Improve Facility Selection/10-10 Health Apps-10-10CG-[date].md

```diff
@@ -2,71 +2,90 @@
 
 ## Instructions
 
-Your document should be brief and high-level.  Please keep it to a single page.  Focus on the high level and link to supporting material where appropriate; this is _not_ a detailed engineering spec.
+Your document should be brief and high-level. Please keep it to a single page. Focus on the high level and link to supporting material where appropriate; this is _not_ a detailed engineering spec.
 
-Some of the items below may not apply to your work--that's okay.  You may not be able to fill in some items that _do_ apply to your work--that's also okay.  If you don't have answers, please come ready to ask questions.
+Some of the items below may not apply to your work--that's okay. You may not be able to fill in some items that _do_ apply to your work--that's also okay. If you don't have answers, please come ready to ask questions.
 
 ## Checklist
 
 - Product description
-    + Brief overview of motivation for the change from an engineering & security point of view
-    + [Link to Collaboration Cycle Request issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/51980)
+  - Brief overview of motivation for the change from an engineering & security point of view
+  - [Link to Collaboration Cycle Request issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/51980)
 - UX design description
-    + For user-facing changes, link to UX prototype or wireframes if available
-         + [Figma designs](https://www.figma.com/design/TxXD5bGUOhbHHWLb85GPjK/10-10CG?node-id=2478-18708&node-type=SECTION&t=LIbnyXIFEnPa5jZz-0)
-    + Call out any engineering challenges; UX is reviewed in the [Design Intent meeting][DI]
+  - For user-facing changes, link to UX prototype or wireframes if available
+    - [Figma designs](https://www.figma.com/design/TxXD5bGUOhbHHWLb85GPjK/10-10CG?node-id=2478-18708&node-type=SECTION&t=LIbnyXIFEnPa5jZz-0)
+  - Call out any engineering challenges; UX is reviewed in the [Design Intent meeting][DI]
 - Frontend changes
-    + Identify any significant code changes
-    + Identify any new design system components needed or changes to current components
-    + Describe any product analytics being gathered
+  - Identify any significant code changes
+    - New `FacilitySearch.jsx` custom component that uses the `VaSearchInput` component and makes a request to the mapbox api with search criteria (zip code, city, state, or full address) and returns lat/long coordinates that are used to make a reuqest to the new caregivers facilities endpoint in `vets-api` that calls the Lighthouse facilities api from `vets-api`.
+    - All changes are within the `caregivers` directory
+  - Identify any new design system components needed or changes to current components
+    - None
+  - Describe any product analytics being gathered
```

Comment:
- Total applications being rerouted (source: stakeholder provided)
      - Total Applications being submitted (source: [Datadog](https://vagov.ddog-gov.com/dashboard/p5g-fys-epz/1010-health-apps?fromUser=false&fullscreen_end_ts=1730229097128&fullscreen_paused=false&fullscreen_refresh_mode=sliding&fullscreen_section=overview&fullscreen_start_ts=1730225497128&fullscreen_widget=1652960129845848&refresh_mode=sliding&from_ts=1730224696206&to_ts=1730228296206&live=true))
      - Facility Selection page - avg engagement time (source: [Google Analytics](https://analytics.google.com/analytics/web/#/analysis/p419143770/edit/8un9wi5nSN-WZ-KqWHHBPA))
      - New Facility Confirmation page  - avg engagement time (source: [Google Analytics)](https://analytics.google.com/analytics/web/#/analysis/p419143770/edit/8un9wi5nSN-WZ-KqWHHBPA)

Reply:
I added these in 👍